src/std/build.ss

```diff
@@ -5,7 +5,10 @@
 (def libdir (path-expand "lib" (getenv "GERBIL_BUILD_PREFIX" (gerbil-home))))
 (add-load-path srcdir)
 
-(eval '(import :std/gambit-sharp))
+(let (gambit-sharp
+      (path-expand "lib/_gambit#.scm"
+                   (getenv "GERBIL_BUILD_PREFIX" (gerbil-home))))
+  (eval `(begin-foreign (include ,gambit-sharp))))
```

Comment:
Do we always need it all the time now?

Reply:
only for the std build, which is the sole reason gambit-sharp ended existing.